app/experimenter/static/js/components/LabeledMultiSelect.js

```diff
@@ -0,0 +1,115 @@
+import { boundClass } from "autobind-decorator";
+import PropTypes from "prop-types";
+import React from "react";
+import { Row, Col, FormLabel } from "react-bootstrap";
+import Select from "react-select";
+
+import Error from "experimenter/components/Error";
+import HelpBox from "experimenter/components/HelpBox";
+
+@boundClass
+class LabeledMultiSelect extends React.PureComponent {
+  static propTypes = {
+    as: PropTypes.string,
+    children: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.node),
+      PropTypes.node,
+    ]),
+    dataTestId: PropTypes.string,
+    error: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.array,
+      PropTypes.object,
+    ]),
+    helpContent: PropTypes.object,
+    id: PropTypes.string,
+    index: PropTypes.number,
+    labelColumnWidth: PropTypes.number,
+    label: PropTypes.string,
+    name: PropTypes.string,
+    note: PropTypes.string,
+    onChange: PropTypes.func,
+    rows: PropTypes.string,
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    type: PropTypes.string,
+    optional: PropTypes.bool,
+    options: PropTypes.array,
+    placeholder: PropTypes.string,
```

Comment:
Looks like we only pass some of these into the Select down below, does it get cranky if you don't specify all of these?  Can you specify only the ones that we pass into the Select?

Reply:
I took out the ones that weren't actually applicable here. I probably started with a big list copied from another component's prop types and didn't take out all the inapplicable ones.